### ISSUE TITLE
Issue building using maven

### ISSUE THREAD

**Issue by sdk29** [thulab/iotdb-benchmark#85]
I'm following the Quick Start instructions in the README.md file.

When I run the command
```
mvn clean package -Dmaven.test.skip=true
```

I get an error that the iotdb-session artifact could not be found.
```
Could not find artifact org.apache.iotdb:iotdb-session:jar:0.9.0-SNAPSHOT
```

I was unable to find any proper dependency related to iotdb-session. Can you help resolve this issue?

OS is Ubuntu 16.04

**Comment by liuruiyiyang** [comment]
```iotdb-session``` is newly introduced in Apache IoTDB v0.9.0(which has not been released yet). To install ```iotdb-session``` on your system, you can try this:

```git clone https://github.com/apache/incubator-iotdb.git```

```cd incubator-iotdb```

```mvn clean install -DskipTests```

**Comment by liuruiyiyang** [comment]
Hi, consider the ```iotdb-session``` feature is not released, we change the iotdb version to v0.8.0 now, you can pull the latest master branch and build again.
